### PR TITLE
feat: send the whole payload with first message

### PR DESCRIPTION
**Pull request by Cifko**
Description
---
Send whole payload with first message, not just the id.

### CODE REVIEW

**Review by stringhandler — COMMENTED** [review]
Looks like the right track. It seems to include the payload in every node, not just the first one though

**Comment by Cifko** [comment]
> Looks like the right track. It seems to include the payload in every node, not just the first one though

It should be fixed now.

**Review by sdbondi — COMMENTED** [review]
utACK

File: applications/tari_validator_node/proto/dan/consensus.proto [hunk 1]
```diff
@@ -44,14 +44,15 @@ message QuorumCertificate {
...
-  bytes payload = 2;
+  bytes payload_id = 2;
   uint64 height = 3;
   bytes shard = 4;
   uint64 payload_height = 5;
   repeated ObjectPledge local_pledges = 6;
   uint64 epoch = 7;
   bytes proposed_by = 8;
   QuorumCertificate justify = 9;
+  optional TariDanPayload payload = 10;
```
Comment (sdbondi) [inline applications/tari_validator_node/proto/dan/consensus.proto]: this is proto2 syntax, in proto3 all fields are optional

**Review by sdbondi — APPROVED** [review]

**Comment by sdbondi** [comment]
utACK

**Review by stringhandler — APPROVED** [review]
utACK

**Pull request merged**